ar.xjuan.Cambalache.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id" : "ar.xjuan.Cambalache",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "separate-locales": false,
+    "command" : "cambalache",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "lxml",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install lxml-4.6.2-cp38-cp38-manylinux1_x86_64.whl"
```

Comment:
this is x86_64 specific. Please, use the source package instead.

Also I belive lxml is in the GNOME runtime.

Reply:
this is why the build fail on `aarch64`

Reply:
looks like it does, maybe it was not in 38